prac_05/emails.py

```diff
@@ -0,0 +1,57 @@
+### this was hard to do because I am so used to using packages like regex
+
+
+def main():
+    emaildict = {}
+    email = 1
```

Comment:
What is this? Why is email the number 1? Does that make sense? Follow the standard pattern please: https://github.com/CP1404/Starter/wiki/Programming-Patterns#while-loops-indefinite-iteration